src/hooks/useStepFormSubmit.ts

```diff
@@ -22,13 +22,10 @@ export default function useStepFormSubmit<T extends keyof OnyxFormValuesMapping>
     return useCallback(
         (values: FormOnyxValues<T>) => {
             if (shouldSaveDraft) {
-                const stepValues = fieldIds.reduce(
-                    (acc, key) => ({
-                        ...acc,
```

Comment:
Again, I wonder if we could create a lint rule for no-spread-in-reduce?

Reply:
Here is what I found: https://biomejs.dev/linter/rules/no-accumulating-spread/

Reply:
yeah, but that's different from ESLint. So we might need to build a custom ESLint rule